app/views/marketing/index.html.erb

```diff
@@ -4,10 +4,12 @@
   <div class="content-wrapper-narrow">
     <h2><%= t(".heading") %></h2>
     <p class="hero__subheading"><%= t(".subheading") %></p>
-    <form action="">
-      <input type="text">
-      <input type="submit">
-    </form>
+
+    <%= simple_form_for @registration, url: new_registration_path, method: :get do |f| %>
+      <%= f.input :zipcode %>
+
+      <%= f.submit t(".donate") %>
```

Comment:
could you use the i18n simple form helpers instead of explicitly writing in the translation here?